### PR TITLE
feat: added grid

### PR DESCRIPTION
**Pull request by vaibhav-mahalle**
- created two-column and three-column grids.
-
Screenshot (76)
Screenshot (77)

### CODE REVIEW

**Review by tanishq20 — APPROVED** [review]
Good that you have used a consistent class name.

File: Documentation/grid/grid.html [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+    <header class="navigation">
+        <div class="hero-heading">Dexterous</div>
+      </header>
+      <div class="drawer-container">
+        <div class="item-box">
+          <div class="box-pill">
+            <a
+              class="btn btn-link-noBorder"
+              href="/documentation/installation/installation.html"
+            >
```
Comment (iamsaurav28) [inline Documentation/grid/grid.html:27]: try to keep close tag in code only not down,  check it overall looks good

**Review by iamsaurav28 — APPROVED** [review]

File: stylesheets/utilities.css [hunk 2]
```diff
@@ -169,5 +174,20 @@
...
 }
 
-
-
+/* Grids */
+.grid {
+    display: grid;
+    width: 100%;
+  }
+
```
Comment (Asypher00) [inline stylesheets/utilities.css:182]: can remove extra spaces

File: Documentation/grid/grid.html [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+    <h4 class="subheading-highlight">Three column grid</h4>
+    
+    <div class="component-display-div">
+        <div class="grid grid-gap three-col-grid">
+            <div class="grid-container  m-1 flex-center">Left Grid Block</div>
+            <div class="grid-container  m-1 flex-center">Center Grid Block</div>
+            <div class="grid-container  m-1 flex-center">Right Grid Block</div>
+        </div>
+    </div>
+
```
Comment (Asypher00) [inline Documentation/grid/grid.html:174]: can remove extra spaces

File: Documentation/grid/grid.html [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+    <pre>
+        <code class="highlight-code">
+<div class="grid grid-gap two-col-grid">
+    <div class="grid-container  m-1 flex-center">Left Grid Block</div>
+    <div class="grid-container  m-1 flex-center">Right Grid Block</div>
+</div>
+        </code>
+    </pre>
+
```
Comment (Asypher00) [inline Documentation/grid/grid.html:163]: can remove extra spaces

**Review by Asypher00 — COMMENTED** [review]
Overall the code looks good and neat!

**Pull request merged**